cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -459,6 +476,21 @@ def qubits(self) -> Tuple[raw_types.Qid, ...]:
         return tuple(sorted(qs))
 
     def with_qubits(self, *new_qubits: 'cirq.Qid') -> 'PauliSum':
+        """Return a new PauliSum on `new_qubits`.
+
```

Comment:
I think we can use some example here / highlight that `self.qubits` will give a sorted list of all qubits used in the pauli sum; and new qubits will be mapped 1:1 with the sorted qubits. 

The fact that pauli sum will always return a sorted list of qubits is tricky and could be helpful to provide some examples.

Reply:
Were you thinking maybe a code snippet ?